charts/kargo/templates/controller/configmap.yaml

```diff
@@ -15,6 +15,9 @@ data:
   {{- if .Values.kubeconfigSecrets.kargo }}
   KUBECONFIG: /etc/kargo/kubeconfigs/kubeconfig.yaml
   {{- end }}
+  {{- if .Values.controller.sharedCredentials.enabled }}
+  KARGO_SHARED_CREDENTIALS_NAMESPACES: {{ .Values.controller.sharedCredentials.namespaces }}
```

Comment:
For better or worse, we don't prefix env vars with `KARGO_`, so let's change this to `SHARED_CREDENTIALS_NAMESPACES`, please.

Reply:
I wonder if we should differentiate between what I'm calling "global" credentials and a different/future feature to share credentials between projects, not at the global scope. 

So I'm proposing we name this `GLOBAL_CREDENTIALS_NAMESPACES` since that's what this PR will enable.